src/components/book.jsx

```diff
@@ -1,19 +1,44 @@
 import PropTypes from 'prop-types';
+import { FaCircleNotch } from 'react-icons/fa';
 
 function Book({ bookTitle, author, onClick }) {
   return (
-    <div>
-      <div>
-        <p>Action</p>
-        <h2>{bookTitle}</h2>
-        <p>{author}</p>
+    <div className="book-container">
+      <div className="book-info">
+        <p className="action">Action</p>
+        <h2 className="book-title">{bookTitle}</h2>
+        <p className="author">{author}</p>
         <div>
-          <button type="button" onClick={onClick}>
+          <button type="button" className="button comment">
+            comments
+          </button>
+          <button type="button" className="button" onClick={onClick}>
             remove
           </button>
+          <button type="button" className="button">
+            edit
+          </button>
+        </div>
```

Comment:
- [ ] KIndly increase the spacing between the actions buttons to match the zeplin design 💯 

Zeplin design 👇🏽 
![image](https://github.com/mershark/bookstore/assets/63473584/a72bff1b-d37c-49d4-a286-9fefa96ca6b1)

your design 👇🏽 
![image](https://github.com/mershark/bookstore/assets/63473584/dac79b00-d83c-420c-9c1a-1f6fa71d1a42)